package.json

```diff
@@ -539,7 +539,9 @@
               "build",
               "configure",
               "install",
-              "test"
+              "test",
+              "clean",
+              "clean rebuild"
```

Comment:
Since it's an enum, I would suggest not having a space in the name and use `cleanRebuild` instead.